docs/docs/guides/integrate/logout.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Logout
+---
+
+This guide shows you different concepts and use cases for logout and how to use it in ZITADEL.
+
+## OpenID Connect Single Logout 
+
+### Single Sign On (SSO) vs Single Logout (SLO)
+
+Single Sing On (SSO) allows a user to login once and be authenticated to several related applications.
+Single Logout (SLO) is the counterpart to this. With SLO a user can logout and terminate all related sessions, without actively logout from all systems.
+
+### OpenID Connect Logout
+
+The purpose of a logout is to terminate a user session. 
+Depending on how the session handling is implemented, there are different mechanisms that can be used.
+There are two different possibilities:
+- User Agent stores session information (e.g. in a cookie)
+- Server Side stores session information (e.g. in a database)
+
+OpenID Connect defines three logout mechanisms to address the different architectures:
+- [OpenID Connect Session Management 1.0](https://openid.net/specs/openid-connect-session-1_0.html)
+- [OpenID Connect Front-Channel Logout 1.0](https://openid.net/specs/openid-connect-frontchannel-1_0.html)
+- [OpenID Connect Back-Channel Logout 1.0](https://openid.net/specs/openid-connect-backchannel-1_0.html)
+
+#### Session Management
+
+Session Management in OpenID Connect defines a mechanism for a client (Relying Party, RP) to monitor the login status of the user in the provider (OP, e.g ZITADEL).
+When a user logs out of the provider, the user's session is terminated.
+
+#### RP initiated Logout
+
+All logout mechanisms are triggered by a request from the client.
+
+#### Front-Channel Logout
+
+The user agent handles the front-channel logout. 
+Each client with an OpenID Session of the user that supports front-channel renders an iframe so the logout request is performed on all clients parallel.
+
+:::note
+This is not yet implemented in ZITADEL
+:::
+
+#### Back-Channel Logout
+
+The back-channel logout is a mechanism on the server-side and the user agent does not have to do anything.
+The user will logout from all clients even in the case the user agent was closed.
+
+:::note
+This is not yet implemented in ZITADEL
+:::
+
+## Use Cases
+
+1. Logout all users from current user-agent/browser (current implementation of ZITADEL end_session_endpoint)
+2. Logout my user from current user-agent/browser
+3. Logout my user from all devices
+
+## Logout in ZITADEL
+
+ZITADEL generates a user-agent cookie as soon as a user authenticates the first time. 
+All open sessions on that user-agent (browser) will be stored to the same cookie.
```

Comment:
Could you describe what that implies? Not sure what I should learn from this info.